recipe/meta.yaml

```
@@ -16,7 +16,7 @@ source:
 # number is propagated into the outputs below
 build:
   number: 0
-  skip: true  # [py<=37 and win]
+  skip: true  # [py<=38 and win]
```

I think should be just:
```suggestion
  skip: true  # [py<=38]
```